### PR TITLE
Honeybee Earthtube - With updated Honeybee Honeybee and Run Energy Simulation

### PR DESCRIPTION
**Pull request by antonszilasi**
Post Processing of Earthtube results to follow

### CODE REVIEW

File: src/Honeybee_ Run Energy Simulation.py [hunk 1]
```diff
@@ -917,15 +948,17 @@ def writeBatchFile(self, workingDir, idfFileName, epwFileAddress, EPDirectory =
...
         batchfile = open(batchFileAddress, 'w')
         batchfile.write(batchStr)
         batchfile.close()
         
         #execute the batch file
-        os.system(batchFileAddress)
-        #os.system('C:\\honeybee\\runIt.bat')
+        os.system(batchFileAddress) 
+
+        os.system('C:\\honeybee\\runIt.bat')
```
Comment (mostaphaRoudsari) [inline src/Honeybee_ Run Energy Simulation.py]: @antonszilasi you need to use Honeybee's and project's default folder and not c:\honeybee. This is a bug.

File: src/Honeybee_ Run Energy Simulation.py [hunk 2]
```diff
@@ -1254,11 +1298,17 @@ def main(north, epwFileAddress, EPParameters, analysisPeriod, HBZones, HBContext
                     if schedule != None:
                         if schedule.upper() not in EPScheduleCollection: EPScheduleCollection.append(schedule)
                     else: needToWriteMixSched = True
-            if zone.mixAir == True:
```
Comment (mostaphaRoudsari) [inline src/Honeybee_ Run Energy Simulation.py]: Looks like you are overwriting part of @chriswmackey's code? Chris can check and confirm. @antonszilasi you probably need to sync with master again and check the diff.

File: src/Honeybee_Honeybee.py [hunk 3]
```diff
@@ -1199,9 +1199,9 @@ def objToRAD(self, objFile):
             #mapFile = objFile.replace(".obj", ".map")
             #with open(mapFile, "w") as mapf:
             #    mapf.write(self.matName + " (Object \"" + self.matName + "\");")
-            #cmd = "c:\\radiance\\bin\\obj2rad -f -m " + mapFile + " " + objFile + " > " +  radFile
+            #cmd = "c:\\radiance\\bin\\obj2rad -m " + mapFile + " " + objFile + " > " +  radFile
             
-            cmd = "c:\\radiance\\bin\\obj2rad -f " + objFile + " > " +  radFile
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Honeybee.py]: The same here- You are overwriting some of the changes that I made.

File: src/Honeybee_Honeybee.py [hunk 4]
```diff
@@ -1502,21 +1502,10 @@ def writeTestPtFile(self, subWorkingDir, radFileName, numOfCPUs, analysisRecipe)
     
         if numOfCPUs > numOfPoints: numOfCPUs = numOfCPUs
     
-        if numOfCPUs > 1:
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Honeybee.py]: and even more...

File: src/Honeybee_Select by Type.py [hunk 5]
```diff
@@ -19,7 +19,7 @@
         surfaces: Output surfaces as Grasshopper objects
 """
 ghenv.Component.Name = "Honeybee_Select by Type"
-ghenv.Component.NickName = 'selByType'
+ghenv.Component.NickName = 'selByType_'
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Select by Type.py]: Do we need to add an _ to the end of the name of the component?

File: src/Honeybee_Select by Type.py [hunk 6]
```diff
@@ -61,25 +64,36 @@ def main(HBZones):
...
                     if srf.type == 0 and srf.hasChild:
                         surfaces.append(srf.punchedGeometry)
+                        
+                        HBSurfacestypezero.append(srf) # Added by Anton Szilasi for Honeybee PV generator component
+                        
                     else:
                         surfaces.append(srf.geometry)
-                        
+
+                        otherHBSurfaces.append(srf) # Added by Anton Szilasi for Honeybee PV generator component
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Select by Type.py]: @antonszilasi why don't you ad a new output just for PV? I would name it explicitly after the type. otherHBSurfaces can be really ambiguous.

**Comment by mostaphaRoudsari** [comment]
@antonszilasi thank you for the pull request and great job. I added a couple of comments. Please review and let me know your thoughts. None of them are really hard to address.

File: src/Honeybee_ Run Energy Simulation.py (hunk 2 above)
Comment (chriswmackey) [inline src/Honeybee_ Run Energy Simulation.py]: Yep.  That is my code being overwritten there.

File: src/Honeybee_Select by Type.py (hunk 6 above)
Comment (antonszilasi) [inline src/Honeybee_Select by Type.py]: Yes this is exactly what Ive done now

**Pull request closed without merging**